### PR TITLE
Introduce Referent tags

### PR DESCRIPTION
**Pull request by Remg**
- should be merged with https://github.com/EnMarche/worker/pull/9

### CODE REVIEW

File: src/Admin/AdherentAdmin.php [hunk 1]
```diff
@@ -378,6 +377,20 @@ protected function configureDatagridFilters(DatagridMapper $datagridMapper)
...
+                },
+            ])
+            ->add('referentTags', CallbackFilter::class, [
+                'label' => 'Tags référent',
+                'callback' => function (ProxyQuery $qb, string $alias, string $field, array $value) {
+                    if (!$value['value']) {
+                        return false;
+                    }
+
+                    $value = array_map('trim', explode(',', strtolower($value['value'])));
```
Comment (B-Galati) [inline src/Admin/AdherentAdmin.php]: Use `mb_strtolower` please

File: app/config/security.yml [hunk 2]
```diff
@@ -162,6 +162,9 @@ security:
         ROLE_ADMIN_ADHERENT_TAGS:
             - ROLE_APP_ADMIN_ADHERENT_TAG_ALL
 
+        ROLE_ADMIN_REFERENT_TAGS:
```
[the remark this answers is not in the document]
Comment (Remg) [reply app/config/security.yml]: nice catch.
done!

File: src/Entity/ReferentManagedArea.php [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+     *         @ORM\JoinColumn(name="referent_tag_id", referencedColumnName="id")
+     *     }
+     * )
+     */
+    private $tags;
+
+    /**
+     * @ORM\Column(type="geo_point", nullable=true)
+     */
+    private $markerLatitude;
```
[the remark this answers is not in the document]
Comment (Remg) [reply src/Entity/ReferentManagedArea.php]: indeed. i will confirm that

File: src/Entity/ReferentManagedArea.php [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+        return $this->markerLatitude;
+    }
+
+    public function setMarkerLatitude(?string $markerLatitude): void
+    {
+        if (!$markerLatitude) {
+            $markerLatitude = null;
+        }
+
+        $this->markerLatitude = $markerLatitude;
```
Comment (B-Galati) [inline src/Entity/ReferentManagedArea.php]: As an alternative => `$this->markerLatitude = $markerLatitude ?: null`

File: src/Entity/ReferentTag.php [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+use Doctrine\ORM\Mapping as ORM;
+use Symfony\Bridge\Doctrine\Validator\Constraints\UniqueEntity;
+use Symfony\Component\Validator\Constraints as Assert;
+
+/**
+ * @ORM\Entity(repositoryClass="AppBundle\Repository\ReferentTagRepository")
+ * @ORM\Table(
+ *   name="referent_tags",
+ *   uniqueConstraints={
+ *     @ORM\UniqueConstraint(name="referent_tag_name_unique", columns="name"),
```
Comment (B-Galati) [inline src/Entity/ReferentTag.php]: It's simpler to add the unique constrain on the property with `unique=true`.
cf. https://www.doctrine-project.org/projects/doctrine-orm/en/latest/reference/annotations-reference.html#annref_column

File: src/Entity/ReferentManagedArea.php (hunk 3 above)
[the remark this answers is not in the document]
Comment (Remg) [reply src/Entity/ReferentManagedArea.php]: @illusionOfParadise we will refactor this in another PR to remove useless properties.

**Pull request merged**